godot-core/src/builtin/basis.rs

```diff
@@ -180,6 +182,7 @@ impl Basis {
     /// Returns the scale of the matrix.
     ///
     /// _Godot equivalent: `Basis.get_scale()`_
+    #[doc(alias = "get_scale")]
     #[must_use]
     pub fn scale(&self) -> Vector3 {
```

Comment:
Simple `get_` prefixes don't need to be explicitly annotated, otherwise we have an excessive number of doc aliases.
This applies to all such transforms.

I though this was mentioned in the issue, but saw that's not the case 😕 sorry for that!